### ISSUE TITLE
Duplicate Pony.fm accounts can be created by changing your Poniverse email address

### ISSUE THREAD

**Issue by Feld0** [Poniverse/Pony.fm#29]
Pony.fm's integration with Poniverse.net erroneously [uses email addresses](https://github.com/Poniverse/Pony.fm/blob/master/app/Http/Controllers/AuthController.php#L100) to determine whether a Poniverse user already has a Pony.fm account.

Changing the email address on your Poniverse account can thus result in a second Pony.fm account being created with the same display name - which, among other issues, creates a conflicting slug for profile URL's.

cc @Lavoaster

**Comment by Lavoaster** [comment]
Does this also mean PFM doesn't implement the Poniverse Pingback?

**Comment by Feld0** [comment]
@Lavoaster Nope - it doesn't. A pingback endpoint [was never added to the routes file](https://github.com/Poniverse/Pony.fm/commits/master/app/Http/routes.php).

**Comment by Feld0** [comment]
[This came up again.](https://mlpforums.com/topic/144702-my-latest-song-not-showing-up-on-profile-issue-with-quotation-marks/#entry4297089)

**Comment by Feld0** [comment]
I ran some queries earlier to get Pony.fm's usernames up to date with Poniverse's (what is now the `username` column used to be the `mlpforums_display_name` column, which often changed over time) but that didn't remove the actual duplicate accounts.

Had been meaning to write the script to finish that for some time but that didn't happen until this morning. That's now done, the script has been run, and a bunch of duplicate accounts have now been merged. There is only one duplicate account left in the database that wasn't caught by all of this, which I'll fix manually.
